tests/measurements/test_counts.py

```diff
@@ -291,11 +404,54 @@ def circuit():
 
         custom_measurement_process(dev, spy)
 
+    @pytest.mark.all_interfaces
+    @pytest.mark.parametrize("interface", ["autograd", "jax", "tensorflow", "torch"])
+    def test_batched_counts_operator_finite_shots(self, interface, mocker):
```

Comment:
Can you add this file in the new return type tests? There enable_return is true. We need all tests file to be there as well, good occasion to do it :+1: